cyberbrain/value_stack.py

```diff
@@ -155,7 +155,15 @@ def copy(self) -> SymbolWithCustomValueStackItem:
         )
 
     def __repr__(self) -> str:
-        return "SymbolWithCustomValueStackItem{" + repr(self.custom_value) + "}"
+        return (
```

Comment:
why not f string?

Reply:
Using f-strings are just as ugly cause of the extra curly braces, you can see it in the new version